queries.yaml

```diff
@@ -239,6 +239,20 @@ post_process:
       items_matching:
         kind: station
       rank_key: tile_rank
+  - fn: TileStache.Goodies.VecTiles.transform.rank_features
+    params:
+      source_layer: places
+      items_matching:
+        kind: [neighbourhood, microhood, macrohood]
+      rank_key: kind_tile_rank
+      start_zoom: 11
+  # not sure if we want to drop n_photos or not, might be useful for
+  # testing in the interim
+  # - fn: TileStache.Goodies.VecTiles.transform.drop_features_where
```

Comment:
This is a bit confusing; it's dropping the property, but not the feature, because `n_photos` is never identically `True`. Do we need a separate function for just dropping properties, or perhaps just improve `drop_features_where` to be clearer about what's going on? Perhaps that's a separate issue anyway...